eva_sub_cli/validators/native_validator.py

```diff
@@ -0,0 +1,62 @@
+import os
+import subprocess
+
+from ebi_eva_common_pyutils.command_utils import run_command_with_output
+from ebi_eva_common_pyutils.logger import logging_config
+
+from eva_sub_cli.validators.validator import Validator
+
+logger = logging_config.get_logger(__name__)
+
+
+class NativeValidator(Validator):
+
+    def __init__(self, mapping_file, output_dir, metadata_json=None, metadata_xlsx=None,
+                 vcf_validator_path='vcf_validator', assembly_checker_path='vcf_assembly_checker',
+                 biovalidator_path='biovalidator', submission_config=None):
```

Comment:
Do you anticipate that the executable paths could be set by the parent maybe from command line args ?